src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Content Fragments workflows
+description: Explore enhanced functionality in the Content Fragments Console and AEM Content Fragment Editor, simplifying workflow initiation.
+contributors:
+- AdobeDocs/uix
+---
+
+# Content Fragments workflows
+
+This extension provides UI extensibility, allowing customers to initiate an AEM workflow for selected content fragments.
```

Comment:
I think the extension doesn't really provide UI extensibility. It mostly uses it

Reply:
Rephrased